project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -527,6 +527,10 @@ function c4m_user_og_c4m_membership_cta($group) {
   $access = c4m_og_get_access_type($group);
 
   if (og_is_member('node', $group->nid, 'user', NULL, array(OG_STATE_PENDING))) {
+    $message = 'This is a moderated group. Your request is waiting for approval. Continue to explore group %group content';
+    $replacements = array('%group' => $group->title);
+    drupal_set_message(t($message, $replacements));
```

Comment:
@DavidHernandez 
not sure the sniffer will like it, you'll probably need to move the text itself to the `t()` instead of the `$message`.